int8 standby_mode
---
bool result